src/app/components/Application/Application.jsx

```diff
@@ -70,12 +69,9 @@ class Application extends React.Component {
         });
       }
     });
-    const style = {
-      xtrasmallBreakPoint: '483px',
-    };
-    const mediaMatcher = window.matchMedia(`(max-width: ${style.xtrasmallBreakPoint})`);
-    this.checkMedia(mediaMatcher);
-    mediaMatcher.addListener(this.checkMedia);
+
+    window.addEventListener('resize', this.onWindowResize.bind(this));
```

Comment:
wish I could say more about this but it appears to work. Still have the issue of the table on the index page being a little too big for the phone screen, but that isn't new.